core/kits/assets/js/component.js

```diff
@@ -27,6 +27,7 @@ export default class extends $e.modules.ComponentBase {
 		return {
 			menu: () => {
 				elementor.getPanelView().setPage( 'kit_menu' );
+				this.currentTab = 'menu';
```

Comment:
This was causing a bug with the backbutton in the panel, we were basically on the menu tab but the currentTab  was always the last one set like 'global-typography' or 'global-styles' and this was causing an issue with the re-render check in component.js. Every tab should set its own name like what the global tabs do now to avoid inconsistency

Reply:
Talked about it with @Nevoss, waiting for resolve.